packages/manager/apps/web/client/app/private-database/order/clouddb/private-database-order-clouddb.service.js

```diff
@@ -116,8 +116,26 @@ export default class PrivateDatabaseOrderCloudDb {
     );
   }
 
-  static getOrderableRamSizes(schema) {
-    return get(schema.models, 'hosting.PrivateDatabase.AvailableRamSizeEnum')
-      .enum;
+  getOrderedRamSizes(plans) {
+    const sizeRegex = /(\d+)\s*(GB|MB)$/;
+
+    return plans
+      .map((plan) => {
+        const matches = plan.invoiceName.match(sizeRegex);
+        if (!matches) {
+          return null;
+        }
+
+        const value = parseInt(matches[1], 10);
+        const ram = matches[2] === 'GB' ? value * 1024 : value;
+        return {
+          label: this.$filter('bytes')(ram, undefined, false, 'MB'),
+          value: ram,
+        };
+      })
+      .filter((value) => value !== null)
```

Comment:
Can you merge these two loop into one reduce.
And if you feel adventurous include also the sort (but it's more complicated)